app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  before_action :require_user
+
+  def create
+    @like = Like.new(recipe_id: params[:recipe_id])
+    @recipe = Recipe.find(params[:recipe_id])
+    @like.chef = current_chef
+    @like.save
```

Comment:
probably good to "bang" the save so it won't fail silently. In the future, we need an enhancement that will flash and error to the user if the save failed.

Reply:
done!